nist80022.go

```diff
@@ -329,36 +332,36 @@ func BlockFrequencyTest(bits string, blocksize int) (float64, bool) {
 //
 // bits: the input bit string to test.
 func RunsTest(bits string) (float64, bool) {
-	m_seq := []byte(bits)
+	mSeq := []byte(bits)
 
-	m_numOnes := 0
-	for _, bit := range m_seq {
+	mNumOnes := 0
+	for _, bit := range mSeq {
 		if bit == '1' {
-			m_numOnes++
+			mNumOnes++
 		}
 	}
-	pi := float64(m_numOnes) / float64(len(m_seq))
+	pi := float64(mNumOnes) / float64(len(mSeq))
 
-	if math.Abs(pi-0.5) >= 2/math.Sqrt(float64(len(m_seq))) {
+	if math.Abs(pi-0.5) >= 2/math.Sqrt(float64(len(mSeq))) {
 		return 0.0, false
 	}
 
-	V_n := 0
-	for i := 0; i < len(m_seq)-1; i++ {
-		if m_seq[i] != m_seq[i+1] {
-			V_n++
+	Vn := 0
+	for i := 0; i < len(mSeq)-1; i++ {
+		if mSeq[i] != mSeq[i+1] {
+			Vn++
 		}
 	}
-	V_n++
+	Vn++
 
-	numerator := math.Abs(float64(V_n) - 2*float64(len(m_seq))*pi*(1-pi))
-	denominator := 2 * math.Sqrt(2*float64(len(m_seq))) * pi * (1 - pi)
+	numerator := math.Abs(float64(Vn) - 2*float64(len(mSeq))*pi*(1-pi))
+	denominator := 2 * math.Sqrt(2*float64(len(mSeq))) * pi * (1 - pi)
 	if denominator == 0 {
 		return 0.0, false
 	}
-	P_value := math.Erfc(numerator / denominator / math.Sqrt2)
+	PValue := math.Erfc(numerator / denominator / math.Sqrt2)
```

Comment:
If we must use camel case, let stick with the convention of first word or letter is lowercase (camel vs. pascal)